data-hub/src/main/java/com/marklogic/hub/flow/SimpleFlow.java

```diff
@@ -26,8 +26,8 @@
  */
 public class SimpleFlow extends AbstractFlow {
 
-    public SimpleFlow(String name) {
-        super(name, "simple");
+    public SimpleFlow(String domainName, String name) {
```

Comment:
would be better to use "flowName", I think -- also consistent with FlowManagerService.